### PR TITLE
Fixed bugs in re-generating the dataset

### PR DESCRIPTION
**Pull request by KelvinYang0320**
I tried to regenerate the dataset and found a few problems:
1. Create ./data before creating ./data/unique_archs.pickle.
2. Fix parameter mismatches in `train.train_single()`.
3. It seems that ` _get_image_size()` only exists in torchvision<=0.10.
    * @vaenyr What are your PyTorch and torchvision versions?

### CODE REVIEW

**Comment by vaenyr** [comment]
Hi, thanks for catching these errors.

You can check env info (including pytorch and torchvision versions) used to train each individual model in blox-env.csv
From the top of my head it's 1.9.1 and 0.10.1 for the vast majority of models, although there are execptions to that (this basically depends on a server used to train a particular model, in some cases we couldn't enforce the exact same pt version).

Btw, if you want to train with higher pt version you can try to simply use Cutout from torchvision - the code that's causing issues >0.10.0 is basically backport of Cutout from higher torchvision versions with some API adjustments.

File: train_range.py [hunk 1]
```diff
@@ -16,6 +16,8 @@ def get_points(start, end):
             models = pickle.load(f)
     else:
         models = blox.search_space.get_unique_architectures()
+        if not os.path.isdir('data'):
+            os.mkdir('data')
```
Comment (vaenyr) [inline train_range.py:20]:
```suggestion
        os.makedirs(name, exist_ok=True)
```

**Comment by KelvinYang0320** [comment]
> You can check env info (including pytorch and torchvision versions) used to train each individual model in blox-env.csv

@vaenyr  Thank you! I got it.

> Btw, if you want to train with higher pt version you can try to simply use Cutout from torchvision - the code that's causing issues >0.10.0 is basically backport of Cutout from higher torchvision versions with some API adjustments.

I cannot find `Cutout` in torchvision. Do you mean `RandAugment`?

> this basically depends on a server used to train a particular model, in some cases we couldn't enforce the exact same pt version

To run blox without enforcing the version, I think let users use `torchvision.transforms.RandAugment` or `utils.RandAugment ` according to their torchvision version is better.  The current version of blox cannot run with torchvision>0.10.

**Comment by vaenyr** [comment]
Yes, sorry I meant RandAug. And yes, the idea would be to alias utils.RandAugment to torchvision.transforms.RandAugment if it exists in torchvision (>0.10), otherwise use the provided code (<=0.10). I will push a relevant change shortly.

**Comment by vaenyr** [comment]
Sorry! I've pushed changes for that and also made some other minor fixes in the .sh scripts.
Feel free to open any PR you feel is going to be useful, though :) e.g. requirements

**Pull request merged**